client/tests/e2e/specs/test_create_org_modal.ts

```diff
@@ -125,29 +125,23 @@ describe('Create a new organization', () => {
     cy.get('.card').should('have.length', 2);
   });
 
-  // Uncomment when https://github.com/Scille/parsec-cloud/issues/5429 is done.
-
-  // it('Close with X button', () => {
-  //   cy.get('#create-organization-button').click();
-  //   cy.get('.popover-viewport').find('ion-item').first().click();
-  //   cy.get('.create-organization-modal').find('.modal-header__title').contains('Create an organization');
-  //   cy.get('.closeBtn').should('be.visible');
-  //   cy.get('.closeBtn').click();
-  //   cy.get('ion-alert').contains('Are you sure?');
-
-  //   // Cancel
-  //   cy.get('ion-alert').get('.alert-button-role-cancel').click();
-  //   cy.get('ion-alert').should('not.exist');
-  //   cy.get('.create-organization-modal').should('exist');
-
-  //   cy.get('.closeBtn').click();
-  //   cy.get('ion-alert').contains('Are you sure?');
-
-  //   // Confirm
-  //   cy.get('.alert-button-role-confirm').click();
-  //   cy.get('ion-alert').should('not.exist');
-  //   cy.get('.create-organization-modal').should('not.exist');
-  // });
+  it('Close with X button', () => {
+    cy.get('#create-organization-button').click();
+    cy.get('.popover-viewport').find('ion-item').first().click();
+    cy.get('.create-organization-modal').find('.modal-header__title').contains('Create an organization');
+    cy.get('.create-organization-modal').find('.closeBtn').should('be.visible');
+    cy.get('.create-organization-modal').find('.closeBtn').click();
+
+    cy.get('.question-modal-ontop').find('.ms-modal-header__title').contains('Are you sure you want to cancel the process?');
+    cy.get('.question-modal-ontop').find('#cancel-button').click();
+    cy.get('.question-modal-ontop').should('not.exist');
+
+    // Main modal does not dismiss.
```

Comment:
is it normal to have this commented code?

Reply:
Yeah. It basically is the code that was previously commented but up-to-date. I still can't figure out why on Cypress the modal does not dismiss properly, so I'm just keeping it commented in case we figure it out someday.